test/addons/csi-addons/kustomization.yaml

```diff
@@ -3,10 +3,10 @@
 
 ---
 resources:
-  - https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.9.1/deploy/controller/crds.yaml
-  - https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.9.1/deploy/controller/rbac.yaml
-  - https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.9.1/deploy/controller/setup-controller.yaml
+  - https://github.com/csi-addons/kubernetes-csi-addons/releases/download/v0.10.0/crds.yaml
```

Comment:
If we are not pointing to a kustomize file at the URL then we could stay with raw content instead? This does not reduce lines to update here as versions change.

Reply:
But download url is much nicer than raw url - shorter, and more important, use something which is officially part of the release. The result should be the same, github probably redirect to the same file.

Reply:
Ah ok you are pointing to the download, I thought you were again going with the git clone

Reply:
I wish we had kustomization upstream so we can do the git clone :-)